build.gradle

```diff
@@ -672,6 +672,7 @@ project(':streams') {
   archivesBaseName = "kafka-streams"
 
   dependencies {
+    compile project(':core')
```

Comment:
@mjsax I'm afraid we are not supposed to introduce a kafka core dependency on kafka streams.